admin-manual/installation/linux/ubuntu-bionic.rst

```diff
@@ -594,100 +352,385 @@ installation or you can change the user used by AtoM in :ref:`config.php <config
-information on these settings see: :ref:`customization-config-files`.
+for example, the default timezone and culture of the application. Depending on
+your local requirements, it may be preferable to configure some of these now.
+For more information on these settings see: :ref:`customization-config-files`.
 
 .. _linux-ubuntu-bionic-security-considerations:
 
 Security considerations
 =======================
 
-Now that AtoM is installed, please take a moment to read our
-:ref:`security section <security>` where we will show you how to
-configure the firewall in Ubuntu and back up AtoM.
+Now that AtoM is configured and installed, please take a moment to read our
+:ref:`security section <security>` where we will show you how to configure
+the firewall in Ubuntu and other options in AtoM configuration.
 
 We strongly encourage our users to configure a firewall because some of the
 services configured should not be exposed in the wild, e.g. Elasticsearch was
 not designed to be accessible from untrusted networks and it's a common attack
 vector.
 
+.. _linux-ubuntu-bionic-serve:
+
+Serve
+=====
+
+There are many web servers out there capable of working well with
+`PHP <http://php.net/>`__. `Apache <https://httpd.apache.org/>`__ is probably
+the most popular and we like it, but we've found that
+`Nginx <http://nginx.com/>`__ adapts itself much better to limited resource
+environments while it also scales better and more predictably under high loads.
+You are welcome to try other solutions, but the following documentation will
+focus on Nginx and PHP-FPM.
+
+Additionally, AtoM requires the configuration of a Gearman worker service.
+
+.. _linux-ubuntu-bionic-filesystem-permissions:
+
+Filesystem permissions
+----------------------
+
+By default, Nginx runs as the www-data user. There are a few directories under
+AtoM that must be writable by the web server. The easiest way to ensure this is
+to update the owner of the AtoM directory and its contents by running:
+
+.. code-block:: bash
+
+   sudo chown -R www-data:www-data /usr/share/nginx/atom
+
+If you are deploying AtoM in a shared environment we recommend you to pay
+attention to the permissions assigned to **others**. The following is an
+example on how to clear all mode bits for others:
+
+.. code-block:: bash
+
+   sudo chmod o= /usr/share/nginx/atom
+
+.. _linux-ubuntu-bionic-workers:
+
+Deployment of workers
+---------------------
+
+Gearman is used in AtoM to support asynchronous tasks, some of which are
+core functionality such as updating the :term:`publication status` of a
+descriptive hierarchy, moving descriptions to a new :term:`parent record`,
+and much more. A worker is just a CLI task that you can run in a terminal
+or supervise with specific tools like upstart, supervisord or systemd.
+The worker will wait for jobs that are assigned by the job server.
+
+We will use systemd to manage the AtoM worker, create the following service
```

Comment:
Minor nitpick: the comma between worker and create should be a period, or a semi-colon.